src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,225 @@
+import * as React from 'react';
```

Comment:
import React, {useState} ....

Reply:
I changed from mui/joy to mui library and concept. I have added the changes for approval.